### ISSUE TITLE
No handling for HTML element extension

### ISSUE THREAD

**Issue by cpboyd** [vaadin/gwt-api-generator#63]
An example of this is iron-form, which extends the default form element.

The generated IronForm widget doesn't function properly.
It specifies an `<iron-form>` tag instead of `<form is="iron-form">`

Moreover, the get/setAction and get/setMethod are not inherited.

**Comment by manolo** [comment]
Yeah, that is something not considered because we expect all custom elements being new tags.
Will discuss with the team and consider this in a new version.
